client/src/app/site/pages/meetings/modules/poll/components/entitled-users-table/entitled-users-table.component.html

```diff
@@ -18,14 +18,36 @@
                         </div>
 
                         <!-- Delegation -->
-                        <div *ngIf="entry.vote_delegated_to_user_id">
+                        <div *ngIf="entry.vote_delegated_to_user_id && !entry.delegation_user_merged_into_id">
                             <span>
                                 ({{ 'represented by' | translate }} {{ entry.vote_delegated_to.getShortName().trim() }})
                             </span>
                         </div>
+                        <div *ngIf="entry.vote_delegated_to_user_id && entry.delegation_user_merged_into_id">
+                            <span>
+                                {{ entry.delegation_user_merged_into }}
+                            </span>
+                        </div>
+                    </div>
+                </div>
+                <div *ngIf="!entry.user && entry.user_merged_into_id">
+                    {{ entry.user_merged_into }}
+                    <div class="user-subtitle">
+                        <!-- Delegation -->
+                        <div *ngIf="entry.vote_delegated_to_user_id && !entry.delegation_user_merged_into_id">
+                            <span>
+                                ({{ 'represented by' | translate }}
+                                {{ entry.vote_delegated_to?.getShortName().trim() }})
```

Comment:
`getShortName` already trims its output. Therefore change to:
```suggestion
                                {{ entry.vote_delegated_to?.getShortName() }})
```

Reply:
done, changed at two places.